lib/std/fs/file.zig

```diff
@@ -927,6 +927,12 @@ pub const File = struct {
     /// Reads all the bytes from the current position to the end of the file.
     /// On success, caller owns returned buffer.
     /// If the file is larger than `max_bytes`, returns `error.FileTooBig`.
+    /// `max_bytes` is not "number of bytes to read", but a maximum.
+    ///
+    /// Note:
+    /// It's imperative to choose a reasonable value for `max_bytes`. The least desirable value to
+    /// use is `@maxInt(usize)`, as it will read until there is no more memory left, and must
+    /// only be used for prototyping.
```

Comment:
`@maxInt()` builtin does not exist and using "must" here is misleading.

Reply:
Should have been `std.math.maxInt`. "Must" was used because of the strong feelings people had in our conversation but I've changed it to "should". Thank you!